foundry.md

```diff
@@ -295,9 +295,28 @@ However, there is no change on the state and therefore this rule just skips the
          <localMem> LM </localMem>
       requires CHEAT_ADDR ==Int #address(FoundryCheat)
        andBool #asWord(#range(LM, ARGSTART, 4)) ==Int 3327641368 // selector ( "label(address,string)" )
+    [priority(40)]
 ```
 
+#### `addr` - Sets a label for a given address.
```

Comment:
The description does not seem to match the implementation

Reply:
Fixed.